add_catch(test_redundant_connection test.cpp)
